tests/run-make/naked-symbol-visibility/rmake.rs

```diff
@@ -0,0 +1,80 @@
+// @only-x86_64
```

Comment:
```suggestion
//@ only-x86_64
```

ugh. I think that should do it? this [failed](https://productionresultssa5.blob.core.windows.net/actions-results/f65dcc02-c8d6-4e60-80e6-f380e1f610b5/workflow-job-run-33594ece-e1ec-5f7a-bb60-02ca082e89d6/logs/job/job-logs.txt?rsct=text%2Fplain&se=2024-08-04T09%3A23%3A30Z&sig=foRnkiMs08XcWglLmQPTALUquoNxkbXLFH%2F4HK4bUcw%3D&ske=2024-08-04T19%3A29%3A30Z&skoid=ca7593d4-ee42-46cd-af88-8b886a2f84eb&sks=b&skt=2024-08-04T07%3A29%3A30Z&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skv=2024-05-04&sp=r&spr=https&sr=b&st=2024-08-04T09%3A13%3A25Z&sv=2024-05-04) on CI with 

```
2024-08-04T05:40:18.8264900Z Command { cmd: DYLD_LIBRARY_PATH="/Users/runner/work/rust/rust/build/aarch64-apple-darwin/test/run-make/naked-symbol-visibility/rmake_out:/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2/lib:/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage0-bootstrap-tools/aarch64-apple-darwin/release/deps:/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage0/lib:/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2-tools-bin:/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2/lib/rustlib/aarch64-apple-darwin/lib" "/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2/bin/rustc" "-L" "/Users/runner/work/rust/rust/build/aarch64-apple-darwin/test/run-make/naked-symbol-visibility/rmake_out" "-Zshare-generics=no" "a_rust_dylib.rs", stdin: None, drop_bomb: DropBomb { command: "/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2/bin/rustc", defused: true, armed_location: Location { file: "/Users/runner/work/rust/rust/tests/run-make/naked-symbol-visibility/rmake.rs", line: 8, col: 5 } } }
2024-08-04T05:40:18.8270060Z output status: `exit status: 1`
2024-08-04T05:40:18.8270660Z === STDOUT ===
2024-08-04T05:40:18.8270930Z 
2024-08-04T05:40:18.8270990Z 
2024-08-04T05:40:18.8271050Z 
2024-08-04T05:40:18.8271230Z === STDERR ===
2024-08-04T05:40:18.8271800Z error: invalid operand for instruction
2024-08-04T05:40:18.8352660Z ##[error]  --> a_rust_dylib.rs:50:20
2024-08-04T05:40:18.8369030Z    |
2024-08-04T05:40:18.8369490Z 50 |     unsafe { asm!("mov rax, 42", "ret", options(noreturn)) }
2024-08-04T05:40:18.8369980Z    |                    ^
2024-08-04T05:40:18.8370290Z    |
```

Reply:
#125813 strikes again.